substrate/frame/nomination-pools/src/lib.rs

```diff
@@ -2742,6 +2768,28 @@ pub mod pallet {
 			let who = ensure_signed(origin)?;
 			Self::do_adjust_pool_deposit(who, pool_id)
 		}
+
```

Comment:
Maybe add rustdoc to the callable? I find them very useful in the interfaces (e.g. polkadotjs).

Reply:
Good spot, an oversight on my part. Thanks!